README.md

```diff
@@ -2,8 +2,84 @@
 
 Repository containing pipeline for processing spinal cord injury (SCI) patients (both DCM and tSCI) using the [Spinal Cord Toolbox (SCT)](https://github.com/spinalcordtoolbox/spinalcordtoolbox).
 
-Steps:
+Pipeline steps:
 1. DICOM to nii (BIDS) conversion
 2. Processing (spinal cord and lesion segmentation, vertebral labeling)
 3. Quality control (QC) + manual compression level labeling
 4. Lesion metric computation
+
+
+## Dependencies
+
+[Spinal Cord Toolbox v6.4](https://github.com/spinalcordtoolbox/spinalcordtoolbox/releases/tag/6.4)
+[dcm2niix >= v1.0.20220505](https://github.com/rordenlab/dcm2niix?tab=readme-ov-file#install)
+
+## Installation
+
+### SCT Installation
+
+1. Open a new terminal:
+
+Press <kbd>command</kbd> + <kbd>space</kbd> and type `Terminal` and press <kbd>return/enter</kbd>.
```

Comment:
This is only for macos right? (maybe specify)

Reply:
you are right! specified in https://github.com/sct-pipeline/balgrist-sci/pull/2/commits/9100c9fdcbb1cb163c95ba1f5fa1d941add64e8d